include/emds/emds.h

```diff
@@ -154,7 +154,8 @@ int emds_load(void);
  * This function clears the flash area for all previously stored data.
  *
  * @retval 0 Success
- * @retval -ERRNO errno code if error
+ * @retval -ECANCELED errno code if it was called before @ref emds_init
```

Comment:
Can you align the @brief string for this with the behavior? E.g.:

"Erases flash areas allocated for the emergency data storage" (and also align the paragraph next to the brief)

Why? Mentioning erase in the statements helps understand what exactly will happen. Clear could mean that system only removes the data leaving some meta information intact.